src/jackdaw/specs.clj

```diff
@@ -36,6 +36,13 @@
                    ::key?
                    (or ::schema ::schema-filename)]))
 
+;; for a serde that is only used for reading, using the schema from the registry
+(s/def :jackdaw.serde/confluent-avro-reader-serde
+  (s/keys
+    :req-un [::serde-keyword
+             ::key?]
+    :opt-un [::schema
+             ::schema-filename]))
```

Comment:
`::key?` is only needed for the Schema Registry. We can probably reuse `:jackdaw.serde/serde` defined above.